nautobot/extras/plugins/utils.py

```diff
@@ -107,7 +108,16 @@ def load_plugin(plugin_name, settings):
 
     # Update Constance Config and Constance Fieldset
     if plugin_config.constance_config:
-        app_config = {f"{plugin_name}__{key}": value for key, value in plugin_config.constance_config.items()}
+        app_config = {}
+        for key, value in plugin_config.constance_config.items():
+            config_item = value
+            # Enforce ConstanceConfigItem namedtuple
+            if not isinstance(value, ConstanceConfigItem):
+                config_item = ConstanceConfigItem(*value)
+                plugin_config.constance_config[key] = config_item
```

Comment:
Do we actually read from `plugin_config.constance_config` ever again after this point? In other words, is this actually necessary/useful?